site/typedoc.json

```
@@ -3,10 +3,13 @@
   "entryPointStrategy": "expand",
   "cleanOutputDir": true,
   "excludeExternals": true,
+  "excludePrivate": true,
+  "excludeInternal": true,
   "name": "IaSQL documentation",
   "categorizeByGroup": false,
   "disableSources": true,
   "readme": "default",
-  "exclude": ["../src/modules/**/mappers/*", "../src/modules/**/migration/*", "../src/modules/iasql_functions/*", "../src/modules/iasql_platform/*", "../src/modules/interfaces.ts", "../src/modules/subscribers.ts"],
+  "exclude": ["../src/modules/**/mappers/*", "../src/modules/**/migration/*", "../src/modules/interfaces.ts", "../src/modules/subscribers.ts",
+"../src/modules/iasql_functions/**", "../src/modules/iasql_platform/**",  ],
```

to be a valid json shouldn't be like this:
```suggestion
"../src/modules/iasql_functions/**", "../src/modules/iasql_platform/**"]
```